### PR TITLE
[ADD] sale_order_line_date_next_reception

### PR DESCRIPTION
**Pull request by mathieudelva**

### CODE REVIEW

File: sale_order_line_date_next_reception/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    date_next_reception = fields.Date(compute="_compute_date_next_reception", compute_sudo=True)
+
+    def _compute_date_next_reception(self):
+        for line in self:
+            line.date_next_reception = False
+            if not(line.product_id.qty_available) and line.state not in ['done', 'cancel']:
```
Comment (sebastienbeau) [inline sale_order_line_date_next_reception/models/sale_order_line.py]: You should test "line.product_id.qty_available < 0" as the qty available can be negatif

**Comment by bealdav** [comment]
Hi @sebastienbeau @mathieudelva 

Thanks for this contribution. I'm also work on this kind of topic ;-)

But it seems not related to odoo-usability ?

Why don't you share in OCA directly ?

Thanks for your feedback

File: sale_order_line_date_next_reception/models/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                picking_model = self.env["stock.picking"]
+                picking_id = picking_model.search([
+                    ('product_id', '=', line.product_id.id),
+                    ("picking_type_id.code", "=", "incoming"), 
+                    ('state', 'in', ['ready', 'waiting', 'assigned'])
+                    ])
+                line.date_next_reception = picking_id and picking_id[0].scheduled_date.date()
```
Comment (sebastienbeau) [inline sale_order_line_date_next_reception/models/sale_order_line.py:24]:
```suggestion
                    ], limit=1)
                line.date_next_reception = picking_id.scheduled_date
```

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

**Comment by mathieudelva** [comment]
Hi @sebastienbeau @bealdav 

I moved the module to sale-workflow:
https://github.com/OCA/sale-workflow/pull/3223

**Pull request closed without merging**